### PR TITLE
Issue 1698: Use getObjectRequest when getting objects

### PR DESCRIPTION
**Pull request by pjohans**

### CODE REVIEW

**Comment by cableman** [comment]
Please look-into the Scrutinizer notices.

File: modules/ting/ting.client.inc [hunk 1]
```diff
@@ -142,55 +143,53 @@ function ting_get_objects($ids) {
...
-        // Get objects as it was not local.
-        $objects[$id] = ting_get_object($id);
-      }
+      // Collect ids to getObjectRequest
+      $get_ids[] = $id;
     }
   }
-  // Open search is limited to 50 results per call, so iterate until all results
-  // have been fetched. It has a limit on the size of the query (>187 rec.id=
-  // ORed together seems to break it).
```
Comment (cableman) [inline modules/ting/ting.client.inc]: Is this limit of 50 records not valid any more?

File: modules/ting/ting.client.inc [hunk 2]
```diff
@@ -142,55 +143,53 @@ function ting_get_objects($ids) {
...
-        if (is_array($collection->objects) && count($collection->objects)) {
-          foreach ($collection->objects as $object) {
-            $objects[$object->id] = $object;
-          }
+/**
+ * Do a getObject request.
+ *
+ * @param $ids
+ * @return array
+ * @throws TingClientException
```
Comment (cableman) [inline modules/ting/ting.client.inc]: Missing parameter description.

File: modules/ting/ting.client.inc [hunk 3]
```diff
@@ -142,55 +143,53 @@ function ting_get_objects($ids) {
     }
   }
 
-  // Not all object are searchable, such as relation etc. So to get over this we
-  // split the request into to groups "own id's" and "others". Where the first
-  // is ensured to be searchable.
```
Comment (cableman) [inline modules/ting/ting.client.inc]: Have the data-well changed, so all objects are fetch-able via getObject requests?

Comment (pjohans) [inline modules/ting/ting.client.inc]: if you mean the parameter allObjects then no.

File: modules/ting/ting.client.inc (hunk 1 above)
Comment (pjohans) [inline modules/ting/ting.client.inc]: i'll ask Finn

Comment (pjohans) [inline modules/ting/ting.client.inc]: default limit is 200 manifestations. It can be set in opensearch's configuration

**Comment by DDBdrift** [comment]
https://platform.dandigbib.org/issues/1698 | Closed

**Pull request closed without merging**